coverage/pom.xml

```diff
@@ -66,6 +66,13 @@
             <artifactId>common</artifactId>
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>gov.cms.ab2d</groupId>
+            <artifactId>common</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
```

Comment:
Just wanted to check if this is needed, or if this is something leftover from when you were testing?

Reply:
I believe, I need this